examples/ProceduralWorld/ProceduralWorld.gd

```diff
@@ -106,7 +106,7 @@ func _process(delta):
 
 ## Private Methods
 func _world_to_chunk(world : Vector3) -> Vector3:
-	var chunk := (world / chunk_size).round()
+	var chunk := (world / chunk_size).floor()
```

Comment:
Thanks, but using round was intended as it rounds to the nearest whole number, with halfway cases rounded away from zero. Whereas floor rounds downward, towards negative infinity. If I remember correctly I needed to us round because chunks behaved incorrectly with floor.